blt-project/blt/sites/default.yml

```diff
@@ -0,0 +1,12 @@
+# @file default.yml
+# Site definition file for the default site.
+aliases:
```

Comment:
I'd prefer for the default values to be taken from project.yml. Users can define a separate default.yml if they wish, but for backwards compatibility, let's support the current approach.